apps/ledger-live-desktop/src/renderer/components/DeviceAction/rendering.tsx

```diff
@@ -627,15 +627,15 @@ export function renderError({
         <ErrorIcon size={44} error={error} />
       </Logo>
       <ErrorTitle>
-        <TranslatedError error={error} noLink />
+        <TranslatedError error={(error as unknown) as Error} noLink />
```

Comment:
should TranslatedError be taking a unknown?

Reply:
we should refactor completely the `@ledgerhq/errors` type definitions so that we do not have so resort to these escape hatches